holidays/countries/cabo_verde.py

```diff
@@ -0,0 +1,293 @@
+        "Porto Novo": "PN",
+        "Praia": "PR",
+        "Ribeira Brava": "RB",
+        "Ribeira Grande": "RG",
+        "Ribeira Grande de Santiago": "RS",
+        "São Domingos": "SD",
+        "São Filipe": "SF",
+        "Sal": "SL",
+        "São Miguel": "SM",
+        "São Lourenço dos Órgãos": "SO",
+        "São Salvador do Mundo": "SS",
+        "São Vicente": "SV",
+        "Tarrafal": "TA",
+        "Tarrafal de São Nicolau": "TS",
+    }
+    supported_categories = (OPTIONAL, PUBLIC)
+    supported_languages = ("de", "en_US", "es", "fr", "pt_CV")
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Ano Novo"))
+
+        # Law # 95/V/99 of March 22nd.
+        if self._year >= 2000:
+            # Democracy and Freedom Day.
+            self._add_holiday_jan_13(tr("Dia da Liberdade e Democracia"))
+
+        # National Heroes Day.
+        self._add_holiday_jan_20(tr("Dia da Nacionalidade e dos Heróis Nacionais"))
+
+        # Ash Wednesday.
+        self._add_ash_wednesday(tr("Quarta-feira de Cinzas"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Sexta-feira Santa"))
+
+        # Easter Sunday.
+        self._add_easter_sunday(tr("Páscoa"))
+
+        # Worker's Day.
+        self._add_labor_day(tr("Dia do Trabalhador"))
+
+        # Law # 69/VI/2005 of May 31st.
+        if self._year >= 2005:
+            # International Children's Day.
+            self._add_childrens_day(tr("Dia Mundial da Criança"))
+
+        # Independence Day.
+        self._add_holiday_jul_5(tr("Dia da Independência Nacional"))
+
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("Dia da Assunção"))
+
+        # All Saints' Day.
+        self._add_all_saints_day(tr("Dia de Todos os Santos"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Natal"))
+
+    def _populate_optional_holidays(self):
+        # Holy Thursday.
+        self._add_holy_thursday(tr("Quinta-Feira Santa"))
+
+        # Mother's Day.
+        self._add_holiday_2nd_sun_of_may(tr("Dia das Mães"))
+
+        # Father's Day.
+        self._add_holiday_3rd_sun_of_jun(tr("Dia dos Pais"))
+
+    def _populate_subdiv_br_public_holidays(self):
+        # Law # 93/82 of November 6th.
+        if self._year >= 1983:
+            # Brava Municipality Day.
+            self._add_holiday_jun_24(tr("Dia do Município da Brava"))
+
+    def _populate_subdiv_bv_public_holidays(self):
+        # Law # 93/82 of November 6th.
+        if self._year >= 1983:
+            # Boa Vista Municipality Day.
+            self._add_holiday_jul_4(tr("Dia do Município da Boa Vista"))
+
+    def _populate_subdiv_ca_public_holidays(self):
+        # Law # 93/82 of November 6th.
+        if self._year >= 1983:
```

Comment:
```suggestion
        if self._year >= 1982:
```
Law - November 6, holiday - November 25. :)